common/setup.go

```diff
@@ -78,8 +78,19 @@ func setupCluster(cfg *config.Config) (err error) {
 		}
 	} else {
 		log.Printf("CLUSTER_ID of '%s' was provided, skipping cluster creation and using it instead", cfg.ClusterID)
+
+		if cfg.ClusterName == "" {
+			cluster, err := OSD.GetCluster(cfg.ClusterID)
+			if err != nil {
+				return fmt.Errorf("could not retrieve cluster name: %v", err)
```

Comment:
Technically this message isn't correct. If this fails it means that it couldn't retrieve data on the cluster as a whole, not the Name.